### PR TITLE
Change DXB Token Address

### PR DESCRIPTION
**Pull request by DefiXBet**
Updated DXB SPL Token Address.

## I agree to not ping *anybody* on Discord/Twitter/email about this pull request. Instead I will inquire by posting a new comment in the pull request if needed.
----
PRs are reviewed in bulk and and can take up to **two weeks** to be merged.  

**Please provide the following information for your token.**

Please include change to the `src/tokens/solana.tokenlist.json` file in the PR. 
DON'T modify any other token on the list.

At minimum each entry should have
* Token Address: 8mgeCL7k2cB2KTN8NhD5biqvcbkfrmBogDpYoHmn5cKQ
* Token Name: DefiXBet
* Token Symbol: DXB
* Logo:

* Link to the official homepage of token: DefiXbet.com

our Token Address has been updated. It can be verified that this request is from our team by checking SPL Token address on DefiXBet Whitepaper slide 23

C7D2E2A4-FA73-4EAB-9821-7F7599D5A01E

### CODE REVIEW

**Comment by DefiXBet** [comment]
hello please also update our Coingecko link 
https://www.coingecko.com/en/coins/defixbet

**Comment by ryoqun** [comment]
context: previous pr: https://github.com/solana-labs/token-list/pull/360

File: src/tokens/solana.tokenlist.json [hunk 1]
```diff
@@ -11241,7 +11241,7 @@
     },
     {
       "chainId": 101,
-      "address": "5GG1LbgY4EEvPR51YQPNr65QKcZemrHWPooTqC5gRPBA",
+      "address": "8mgeCL7k2cB2KTN8NhD5biqvcbkfrmBogDpYoHmn5cKQ",
```
Comment (ryoqun) [inline src/tokens/solana.tokenlist.json:11244]: @bartosz-lipinski DXB contacted me and saying they're wanting to change to this new address. there shouldn't be any devastating thing from technical aspect by updating this already-published address (= mint address)? or there should be new entry for  true `DXB` and change of this entry `name` to `DefiXBet Token (obsolete; test-purpose)`?


They said all of already minted tokens (at [5GG1LbgY4EEvPR51YQPNr65QKcZemrHWPooTqC5gRPBA](https://explorer.solana.com/address/5GG1LbgY4EEvPR51YQPNr65QKcZemrHWPooTqC5gRPBA/largest)) was used test-purpose and it won't be circulating actually.

Comment (DefiXBet) [reply src/tokens/solana.tokenlist.json:11244]: We will distribute our new token with this token address 
8mgeCL7k2cB2KTN8NhD5biqvcbkfrmBogDpYoHmn5cKQ

File: src/tokens/solana.tokenlist.json [hunk 2]
```diff
@@ -14917,7 +14917,7 @@
     },
     {
       "chainId": 101,
-      "address": "5GG1LbgY4EEvPR51YQPNr65QKcZemrHWPooTqC5gRPBA",
```
Comment (ryoqun) [inline src/tokens/solana.tokenlist.json:14920]: https://explorer.solana.com/address/5GG1LbgY4EEvPR51YQPNr65QKcZemrHWPooTqC5gRPBA

File: src/tokens/solana.tokenlist.json [hunk 3]
```diff
@@ -14917,7 +14917,7 @@
...
+      "address": "8mgeCL7k2cB2KTN8NhD5biqvcbkfrmBogDpYoHmn5cKQ",
```
Comment (ryoqun) [inline src/tokens/solana.tokenlist.json:14920]: https://explorer.solana.com/address/8mgeCL7k2cB2KTN8NhD5biqvcbkfrmBogDpYoHmn5cKQ/largest

**Comment by ryoqun** [comment]
(on merging this, i confirmed this mint address change request is truthfully orignated from the team member behind DXB).

**Pull request merged**